Nuance Problem with `move_factors_into_sums` Keyword Argument in `@rewrite`

If `;` is used instead of `,` as shown in docstring for `@rewrite`,  an assertion error is triggered:
```julia
julia> using JuMP, MutableArithmetics

julia> model = Model();

julia> @variable(model, x[1:3]);

julia> @rewrite(sum(x[i]^3 for i in 1:3), move_factors_into_sums = false)
((x[1] ^ 3) + (x[2] ^ 3) + (x[3] ^ 3))

julia> @rewrite(sum(x[i]^3 for i in 1:3); move_factors_into_sums = false)
ERROR: LoadError: AssertionError: Meta.isexpr(args[2], :(=), 2) && (args[2]).args[1] == :move_factors_into_sums
Stacktrace:
 [1] var"@rewrite"(__source__::LineNumberNode, __module__::Module, args::Vararg{Any})
   @ MutableArithmetics C:\Users\bbgui\.julia\packages\MutableArithmetics\geMUn\src\rewrite.jl:22
in expression starting at REPL[6]:1
```

--- a/src/rewrite.jl
+++ b/src/rewrite.jl
@@ -5,7 +5,7 @@
 # one at http://mozilla.org/MPL/2.0/.
 
 """
-    @rewrite(expr; move_factors_into_sums = false)
+    @rewrite(expr, move_factors_into_sums = false)
 
 Return the value of `expr`, exploiting the mutability of the temporary
 expressions created for the computation of the result.
@@ -13,6 +13,10 @@ expressions created for the computation of the result.
 If you have an `Expr` as input, use [`rewrite_and_return`](@ref) instead.
 
 See [`rewrite`](@ref) for an explanation of the keyword argument.
+
+!!! info
+    Passing `move_factors_into_sums` after a `;` is not supported. Use a `,`
+    instead.
 """
 macro rewrite(args...)
     @assert 1 <= length(args) <= 2

